app/errors.js

```diff
@@ -8,3 +8,6 @@ exports.databaseError = message => internalError(message, exports.DATABASE_ERROR
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.SERVICE_GARDEN_ERROR = 'service_garden_error';
```

Comment:
It could be more generic so that in the future it can be used with any external API.